tests/specs/benchmarks/overflow00-nooverflow-spec.k

```diff
@@ -122,8 +123,8 @@ _
 andBool #rangeUInt(256, CONTRACT_BAL)
 andBool #rangeUInt(256, CALLEE_BAL) andBool #range(0 <= CD < 1024)
 andBool #rangeAddress(MSG_SENDER) andBool #range(0 <= A0 < 2 ^Int 256 -Int 1)
-    ensures ?N ==Int A0 +Int 1
-
+    //ensures ?N ==Int A0 +Int 1
+    ensures true
```

Comment:
We don't need the comment or ensures clause.

Please don't leave the old version commented out in the proofs, just update them. It will just confuse future people who need to work on these proofs.

Reply:
I removed all the comments.